### PR TITLE
Added slaves field in Executor v2 CRD 

### PR DESCRIPTION
**Pull request by hiteshwani**
## Pull request description 
Created v2 version for adding slaves filed in Executor CRD.
```
 slaves:
   description: Slaves data to run test in distributed environment
   properties:
     image:
       type: string
   required:
   - image
   type: object
````

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Comment by vsukhin** [comment]
Why are we creating executorV2? Because of slave property?

**Comment by hiteshwani** [comment]
Yes, @vsukhin. 
We've discussed this with @dejanzele as well, and we've decided to create a new version, v2, to incorporate the 'slaves' field into the Executor CRD.

**Comment by vsukhin** [comment]
We intriduce new version when we break compaitibility between versions. We don't use minoj versions, but this is definitely minor one

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG ALPINE_IMAGE
-FROM ${ALPINE_IMAGE}
+FROM  alpine:3.18.0
```
Comment (dejanzele) [inline Dockerfile:2]: Can you revert this change so it doesn't break our CI?

File: controllers/executor/executor_controller.go [hunk 2]
```diff
@@ -57,6 +57,6 @@ func (r *ExecutorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 // SetupWithManager sets up the controller with the Manager.
 func (r *ExecutorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&executorv1.Executor{}).
+		For(&executorv2.Executor{}).
```
Comment (dejanzele) [inline controllers/executor/executor_controller.go:60]: I think we can revert to V1

**Review by vsukhin — CHANGES_REQUESTED** [review]
we increase major version only for incompatible changes

**Comment by hiteshwani** [comment]
Closing this PR. I will create new PR to add slave configuration in Executor V1 CRD.

**Pull request closed without merging**